_data/cryptocurrencies.yml

```
@@ -502,6 +502,14 @@ websites:
     regions:
       - br
 
+  - name: NairaEx
+    url: https://nairaex.com/
+    img: nairaex.svg
+    tfa:
+      - totp
+    doc: https://support.nairaex.com/support/solutions/articles/43000335554
+    regions:
+      - ng
```

```suggestion
      - ng

```
There should be a new line between entries.